wherobots/db/connection.py

```diff
@@ -209,6 +215,17 @@ def __execute_sql(self, sql: str, handler: Callable[[Any], None]) -> str:
             "statement": sql,
         }
 
+        if self.__store:
```

Comment:
having thought about this a bit, imo the sdk should ONLY support presigned urls. because without them the user would have to use the rest api to generate temporary aws credentials, and that endpoint is non-public

Reply:
Although I see your point, I think for now it's simpler to keep the SDK non-opinionated.